.github/workflows/docker_publish.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
-          platforms: linux/x86_64,linux/arm64
+          platforms: linux/x86_64
```

Comment:
This is needed to publish multi-arch image

Reply:
Yes, I just use this branch to build a testing image. :)